src/main/java/com/wavefront/topology/ServiceInfo.java

```diff
@@ -0,0 +1,20 @@
+package com.wavefront.topology;
+
+import com.wavefront.datastructures.TagVariation;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Combines all service-related data. For internal use of package classes.
```

Comment:
please improve java-doc

Reply:
and also maybe it should be in another package like datastructures

Reply:
Java-doc improved.
Regarding moving: I put TraceTopology, TraceTopologySanitizer, ServiceInfo in one package for preserving encapsulation principles. These three classes have wide access to each other, but I don't willing to provide the same access level to others.